lib/classes/extensions/box_shadow.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:boilerplate/styles/colors.dart';
+import 'package:flutter/material.dart';
+
+extension ExtendedBoxShadow on BoxShadow {
```

Comment:
Might be a better name to make an standard to call this 
```suggestion
extension BoxShadowExtension on BoxShadow {
```